### PR TITLE
Add howto docs about using temperature compensation and pre-estimated offsets in WBD and add a top-level WBD README

### LINKED ISSUE #50: Document the added variables to use the temperature coefficients
The PR https://github.com/robotology/whole-body-estimators/pull/45 adds the options to use temperature coefficients and offline estimated offset. We will detail the usage of this functions in the wiki in this repo.

### LINKED ISSUE #55: Update README to add references to wholebodydynamics
The README still contains only references `baseEstimatorV1` , and there is no documentation (or link to documentation) of `wholebodydynamics`.

### PR DESCRIPTION
**Pull request by prashanthr05**
- Move documentation from https://github.com/robotology/whole-body-estimators/wiki/How-to-use-the-temperature-coefficients-and-offset-compensation to `docs` folder
- Moved https://github.com/robotology-legacy/codyco-modules/blob/master/doc/force_control_on_icub.md to `docs` folder  
- Moved doxygen documentation of the `wholeBodyDynamcis` device to a folder level README (removed the doxygen documentation to avoid any future divergence of redundant documentations cc @traversaro's wise words)
- Updated the top-level README to link all the above

Fixes #50 
Fixes #55

### CODE REVIEW

**Comment by prashanthr05** [comment]
Relevant links for this must be added when updating the README for https://github.com/robotology/whole-body-estimators/issues/55

**Comment by prashanthr05** [comment]
friendly ping @HosameldinMohamed @fjandrad @traversaro !!

**Review by traversaro — APPROVED** [review]
Ok. Minor comment: can we make sure that this page is linked by the README or by some more central piece of documentation (doxygen?).

**Comment by prashanthr05** [comment]
> Ok. Minor comment: can we make sure that this page is linked by the README or by some more central piece of documentation (doxygen?).

Ok, I will work on that before merging.

**Review by HosameldinMohamed — APPROVED** [review]

**Comment by prashanthr05** [comment]
Ciao guys! 

I updated the overall documentation for `wholeBodyDynamics` device within this PR. I would ask you to re-review it.

For a demo, please see https://github.com/prashanthr05/whole-body-estimators/tree/doc/howto.

Feel free to update the documentation by adding relevant paper references (which is the only thing missing currently).

**Comment by traversaro** [comment]
Thank @prashanthr05 for handling this! If we want to keep the docs in the README (that I think is indeed a good idea) I suggest to remove them from the doxygen blocks, otherwise the two copies of the docs will probably diverge soon.

**Comment by prashanthr05** [comment]
> Thank @prashanthr05 for handling this! If we want to keep the docs in the README (that I think is indeed a good idea) I suggest to remove them from the doxygen blocks, otherwise the two copies of the docs will probably diverge soon.

Looks like I missed porting part of the documentation from the doxygen to the README, relating to the FT Offsets and temperature coefficients.

I will update that and remove the doxygen then.

**Comment by prashanthr05** [comment]
> > Thank @prashanthr05 for handling this! If we want to keep the docs in the README (that I think is indeed a good idea) I suggest to remove them from the doxygen blocks, otherwise the two copies of the docs will probably diverge soon.
> 
> Looks like I missed porting part of the documentation from the doxygen to the README, relating to the FT Offsets and temperature coefficients.
> 
> I will update that and remove the doxygen then.

I've done this in https://github.com/prashanthr05/whole-body-estimators/commit/c91e03625b95a82799d8fe1f8940f3d36120690f

File: devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h [hunk 1]
```diff
@@ -114,210 +114,9 @@ class wholeBodyDynamicsDeviceFilters
...
- *         </action>
- *
- *         <action phase="shutdown" level="2" type="detach" />
- *
- *     </device>
- * </devices>
- * \endcode
+ * 
+ * For a detailed documentation of the parameters used within this device, please see
+ * robotology/whole-body-estimators/devices/wholeBodyDynamics/README.md
```
Comment (traversaro) [inline devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h:119]: This reference is confusing, as if you are in a fork you should refer to the README of that fork, I would simple refer to a README in the same directory of this file.

**Review by traversaro — APPROVED** [review]

File: devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h (hunk 1 above)
Comment (prashanthr05) [reply devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h]: Fixed in https://github.com/robotology/whole-body-estimators/pull/84/commits/52a91d13fcc41230bada87da23948da2c8f099f6

**Comment by prashanthr05** [comment]
@HosameldinMohamed I request you to look at the relevant changes and merge the PR if you approve. Thanks in advance..!

File: doc/howto/useTemperatureCoefficientsAndOffsetCompensationInWholeBodyDynamics.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+To enable the use of temperature compensation and constant offset features in wholebodydynamics device, it is required to add some variables in the config file. The variables can be grouped into three:
```
[the remark this answers is not in the document]
Comment (prashanthr05) [reply doc/howto/useTemperatureCoefficientsAndOffsetCompensationInWholeBodyDynamics.md:3]: I misunderstood/misread this comment earlier. I have addressed this in https://github.com/robotology/whole-body-estimators/pull/84.

**Comment by traversaro** [comment]
Friendly ping @HosameldinMohamed

**Review by HosameldinMohamed — APPROVED** [review]
It looks good to me, thanks @prashanthr05 for taking care of it.

**Comment by prashanthr05** [comment]
Thanks @HosameldinMohamed !

**Pull request merged**